ark_sdk_python/examples/session_monitoring_activites.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime, timedelta
+
+from ark_sdk_python.auth import ArkISPAuth
+from ark_sdk_python.models.ark_profile import ArkProfileLoader
+from ark_sdk_python.models.common import ArkProtocolType
+from ark_sdk_python.models.services.sm import ArkSMGetSession, ArkSMGetSessionActivities, ArkSMSessionsFilter
+from ark_sdk_python.services.sm import ArkSMService
+
+if __name__ == "__main__":
+    isp_auth = ArkISPAuth()
+    isp_auth.authenticate(profile=ArkProfileLoader().load_default_profile())
+    sm: ArkSMService = ArkSMService(isp_auth)
+    search_by = 'startTime ge {start_time_from} AND sessionDuration GE {min_duration} AND protocol IN {protocols}'
+    search_by = search_by.format(
+        start_time_from=(datetime.utcnow() - timedelta(days=30)).isoformat(timespec='seconds'),
+        min_duration='00:00:01',
+        protocols=','.join([ArkProtocolType.DB[0], ArkProtocolType.SSH[0], ArkProtocolType.RDP[0]]),
```

Comment:
I suggest to move to serialize_sm_protocol_type.
I missed it on the last PR sadly